box/src/Box.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { CSSProperties, ReactNode } from 'react';
+import * as React from 'react';
```

Comment:
why did you change it?
I remember some discussion that this should change, but I assumed it should change to `import React from 'react';` where it's not yet working like that, but not the other way around

Reply:
This change was a consequence of updating deps versions via
```
Module 'react' can only be default-imported using the 'allowSyntheticDefaultImports' flag
This module is declared with using 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.
```
.... fixed by using `allowSyntheticDefaultImports`